components/mpas-ocean/src/analysis_members/mpas_ocn_moc_streamfunction.F

```diff
@@ -509,8 +510,8 @@ subroutine ocn_compute_moc_streamfunction(domain, timeLevel, err)!{{{
             end do
             do k = 2, nVertLevels
                mocStreamValLatAndDepthRegionLocal(1, k, iTransect) = &
-                  mocStreamValLatAndDepthRegionLocal(1, k - 1, iTransect) &
-                  + sumTransport(k - 1, iTransect)
+                  mocStreamValLatAndDepthRegionLocal(1, k + 1, iTransect) &
+                  + sumTransport(k + 1, iTransect)
```

Comment:
@vanroekel, we are getting an out-of-bounds error here at `k=nVertLevels`, when it tries to access `k+1`.  This looks like you are changing the MOC calculation from top-down to bottom-up, but:
1. Line 511 should be `do k = nVertLevels-1, 1, -1` 
2. Before that we need to set 
```
mocStreamValLatAndDepthRegionLocal(1, nVertLevels, iTransect) = sumTransport(nVertLevels, iTransect)
```
3. For clarity, this should be in a separate PR that only changes the direction of the moc calculation to bottom-up.
4. These same points apply to two places below that use `k+1` (lines 612, 717)

Reply:
@mark-petersen I thought I Removed these changes a long time ago at the request of @xylar in his initial review.  I agree the calculation should be removed for this PR.  I wonder if in all the different commits it got reintroduced somehow.  Are you able to revert that change?  I can a little later this morning too.

Reply:
I can take care of it, either with a revert or a single additional commit. I'll see where it came in, but fix it regardless.